argilla/src/argilla/records/_io/_datasets.py

```diff
@@ -70,7 +70,7 @@ def to_datasets(records: List["Record"]) -> "HFDataset":
 
     @staticmethod
     def _record_dicts_from_datasets(dataset: "HFDataset") -> List[Dict[str, Union[str, float, int, list]]]:
-        """Creates a dictionaries from a HF dataset that can be passed to DatasetRecords.add or DatasetRecords.update.
+        """Creates a dictionaries from an HF dataset that can be passed to DatasetRecords.add or DatasetRecords.update.
```

Comment:
This a pronounced 'h' so it gets an 'a'.

```suggestion
        """Creates a dictionaries from a HF dataset that can be passed to DatasetRecords.add or DatasetRecords.update.
```